### PR TITLE
Spotlight button

### PR DESCRIPTION
**Pull request by mrSidSat**
Spotlight button implemented

Issue: #1143

### CODE REVIEW

File: src/components/animated/_spotlight.less [hunk 1]
```diff
@@ -0,0 +1,151 @@
+@import "./_variables.less";
```
Comment (shahednasser) [inline src/components/animated/_spotlight.less:1]: This line is unnecessary you can remove it

File: src/components/animated/_spotlight.less [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	background-color: transparent;
+	overflow:visible !important;
+	cursor:pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	outline:none;
+	border-width: 3px; 
+	border-style: solid;
+	z-index: 0;
```
Comment (shahednasser) [inline src/components/animated/_spotlight.less:45]: Remove lines 32, 34, 35, 38, 39, 40, 41, 42, 43, 44, 45 as they are unnecessary.

Also, change `color: black` to `color: @darkText`

File: src/components/animated/_spotlight.less [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+		position: absolute;
+		height:100%;
+		width:50px;
+		top:100px;
+		opacity:0;
+		z-index: -1;
+		left:50%;
+		transform: translateX(-50%);
+		border-radius: 50%;
+		border-radius: inherit;
```
Comment (shahednasser) [inline src/components/animated/_spotlight.less:101]: remove lines 100 and 101

File: src/components/animated/_spotlight.less [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+		}
+	} 
+	.spotlight-colorspecific-mixin("after/before");
+	&:after{             
+		content:"";
+		border-width:3px;
+		border-style: solid;
+		height:100%;
+		width:100%;
+		top:3px;
```
Comment (shahednasser) [inline src/components/animated/_spotlight.less:70]: change top and left to 6px

File: src/components/animated/_spotlight.less [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+		} 
+		&:before{
+			background-color: white;
+			border:1px solid #9c9a9a;
+		}
+	} 
+	.spotlight-colorspecific-mixin("after/before");
+	&:after{             
+		content:"";
+		border-width:3px;
```
Comment (shahednasser) [inline src/components/animated/_spotlight.less:66]: change to 1px

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work. I have added my review below to refactor the code.

File: src/components/animated/_spotlight.less (hunk 2 above)
Comment (mrSidSat) [reply src/components/animated/_spotlight.less:45]: The stack effect won't work if line 45 is not included

Comment (shahednasser) [reply src/components/animated/_spotlight.less:45]: ok you can leave that line

**Comment by mrSidSat** [comment]
Respective Changes are made

File: src/components/animated/_spotlight.less (hunk 3 above)
Comment (shahednasser) [reply src/components/animated/_spotlight.less:90]: @mrSidSat this change hasn't been made

File: src/components/animated/_spotlight.less (hunk 2 above)
Comment (shahednasser) [reply src/components/animated/_spotlight.less]: @mrSidSat you didn't remove line 32 about the padding

File: src/components/animated/_spotlight.less (hunk 3 above)
Comment (mrSidSat) [reply src/components/animated/_spotlight.less:90]: If this line is removed the spotlight that appears won't be circle and the border won't be consistent over the animation

File: src/components/animated/_spotlight.less (hunk 2 above)
Comment (mrSidSat) [reply src/components/animated/_spotlight.less]: it is removed in the latest commit

File: src/components/animated/_spotlight.less (hunk 3 above)
Comment (mrSidSat) [reply src/components/animated/_spotlight.less]: it is removed in the latest commit

**Comment by shahednasser** [comment]
Great work as always thanks!

@all-contributors please add @mrSidSat for code and idea

**Pull request merged**